common/src/main/resources/emails/RejectImageEmail.html

```diff
@@ -0,0 +1,392 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional //EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
```

Comment:
This is a bit long. It's good for now, since we want to send this out. Just a follow up on our conversation to create a ticket for making email customization easier down the line